vizro-core/examples/scratch_dev/app.py

```diff
@@ -1,66 +1,111 @@
 """Dev app to try things out."""
 
-from typing import List
-
-import pandas as pd
-import plotly.graph_objects as go
 import vizro.models as vm
+import vizro.plotly.express as px
 from vizro import Vizro
+from vizro.figures import kpi_card
 from vizro.models.types import capture
+from vizro.tables import dash_ag_grid, dash_data_table
 
-sankey_data = pd.DataFrame(
-    {
-        "Origin": [0, 1, 2, 1, 2, 4, 0],  # indices inside labels
-        "Destination": [1, 2, 3, 4, 5, 5, 6],  # indices inside labels
-        "Value": [10, 4, 8, 6, 4, 8, 8],
-    }
-)
+df = px.data.iris()
 
 
+# Graph
 @capture("graph")
-def sankey(
-    data_frame: pd.DataFrame,
-    source: str,
-    target: str,
-    value: str,
-    labels: List[str],
-) -> go.Figure:
-    """Creates a sankey diagram based on a go.Figure."""
-    fig = go.Figure(
-        data=[
-            go.Sankey(
-                node={
-                    "pad": 16,
-                    "thickness": 16,
-                    "label": labels,
-                },
-                link={
-                    "source": data_frame[source],
-                    "target": data_frame[target],
-                    "value": data_frame[value],
-                    "label": labels,
-                    "color": "rgba(205, 209, 228, 0.4)",
-                },
-            )
-        ]
-    )
-    fig.update_layout(barmode="relative")
-    return fig
+def my_graph_figure(data_frame, **kwargs):
+    """My custom figure."""
+    return px.scatter(data_frame, **kwargs)
+
+
+class MyGraph(vm.Graph):
+    """My custom class."""
+
+    def build(self):
+        """Custom build."""
+        graph_build_obj = super().build()
+        # DO SOMETHING
+        return graph_build_obj
+
+
+# Table
+@capture("table")
+def my_table_figure(data_frame, **kwargs):
+    """My custom figure."""
+    return dash_data_table(data_frame, **kwargs)()
+
+
+class MyTable(vm.Table):
+    """My custom class."""
+
+    pass
+
+
+# AgGrid
+@capture("ag_grid")
+def my_ag_grid_figure(data_frame, **kwargs):
+    """My custom figure."""
+    return dash_ag_grid(data_frame, **kwargs)()
+
+
+class MyAgGrid(vm.AgGrid):
+    """My custom class."""
+
+    pass
+
+
+# Figure
+@capture("figure")
+def my_kpi_card_figure(data_frame, **kwargs):
+    """My custom figure."""
+    return kpi_card(data_frame, **kwargs)()
+
+
+class MyFigure(vm.Figure):
+    """My custom class."""
+
+    pass
+
+
+# Action
+@capture("action")
+def my_action_function():
+    """My custom action."""
+    pass
+
+
+class MyAction(vm.Action):
```

Comment:
Should those examples note move to tests?

Reply:
You're right! I added some tests 👍